stack/core/src/main/java/org/apache/usergrid/corepersistence/asyncevents/SQSAsyncEventService.java

```diff
@@ -184,7 +185,7 @@ public void ack( final List<QueueMessage> messages ) {
     }
 
 
-    @Override
+//    @Override
     public void index( final EntityIdScope entityIdScope ) {
```

Comment:
Seems like we're tearing this method out of the InMemoryAsyncEvenService, why keep it here ?